examples/http-client-mutual-ssl/http_client_mutual_ssl.md

```diff
@@ -4,8 +4,12 @@ Ballerina supports mutual SSL, which is a certificate-based authentication proce
 
 For more information on the underlying module, see the [`http` module](https://lib.ballerina.io/ballerina/http/latest/).
 
+>**Tip:** You may need to change the certificate file path, private key file path, and trusted certificate file path in the code below.
+
 ::: code http_client_mutual_ssl.bal :::
 
-Run the client program by executing the following command.
+Run the client program by executing the command below.
+
+>**Tip:** # As a prerequisite to running the client, start a sample service secured with mutual SSL.
```

Comment:
```suggestion
>** Info:** # As a prerequisite to running the client, start a sample service secured with mutual SSL.
```

Reply:
Please check and update all the places

Reply:
Same as https://github.com/ballerina-platform/ballerina-distribution/pull/3283#discussion_r990543788.

Reply:
Done.